config/secrets.yml

```diff
@@ -29,4 +29,4 @@ test:
 # and move the `production:` environment over there.
 
 production:
-  secret_key_base: <%= ENV["SECRET_KEY_BASE"] %>
+  secret_key_base: <%= defined?(JRUBY_VERSION) ? java.lang.System.getProperty('kaui.secrect_key_base') : ENV['KAUI_SECRET_KEY_BASE'] %>
```

Comment:
* Typo: `secrect_key_base` should be `secret_key_base`.
* Let's also have a default using ENV variable for JRuby (required for Docker/k8s).

```
defined?(JRUBY_VERSION) ? java.lang.System.getProperty('kaui.secret_key_base', ENV['KAUI_SECRET_KEY_BASE']) : ENV['KAUI_SECRET_KEY_BASE']
```